### PR TITLE
Aligning HMI and MOBILE API for pcmStreamCapabilities

### PR DESCRIPTION
**Pull request by hwilli88**
This proposal is to add param `pcmStreamCapabilities` in `UI.GetCapabilities` response in HMI API.

### CODE REVIEW

File: proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Aligning HMI and MOBILE API for pcmStreamCapabilities
+
+* Proposal: [SDL-NNNN](NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md)
+* Author: [Heather Williams](https://github.com/hwilli88/), [Ankur Tiwari](https://github.com/atiwari9/)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core][HMI]
```
Comment (theresalech) [inline proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md]:
```suggestion
* Impacted Platforms: [Core / HMI]
```

File: proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Introduction
+
+This proposal is to add param `pcmStreamCapabilities` in `UI.GetCapabilities` response in HMI API.
+
+## Motivation
+
+`pcmStreamCapabilities` was introduced by PR: https://github.com/smartdevicelink/sdl_core/pull/472 , but this PR missed to add implementation for HMI API. `pcmStreamCapabilities` is present in MOBILE API, but HMI has no way to provide this information to SDL. As a result, SDL always uses default `hmi_capabilities.json` for `pcmStreamCapabilities` in `RAI` response to MOBILE.
```
Comment (theresalech) [inline proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md]:
```suggestion
`pcmStreamCapabilities` was introduced by PR: https://github.com/smartdevicelink/sdl_core/pull/472 , but this PR missed adding implementation for HMI API. `pcmStreamCapabilities` is present in MOBILE API, but HMI has no way to provide this information to SDL Core. As a result, SDL Core always uses default `hmi_capabilities.json` for `pcmStreamCapabilities` in `RAI` response to MOBILE.
```

File: proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Proposed Solution 
+
+Align HMI API with MOBILE API for `pcmStreamCapabilities` by adding this param to HMI API in `UI.GetCapabilities` response.
+
+```
```
Comment (theresalech) [inline proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md]:
```suggestion
```xml
```

File: proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+</function>
+```
+
+## Potential downsides
+
+Author is not aware of any downsides to proposed solution.
+
+## Impact on existing code
+
+* HMI needs to be updated to provide `pcmStreamCapabilities`
```
Comment (theresalech) [inline proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md]:
```suggestion
* HMI needs to be updated to provide `pcmStreamCapabilities`.
```

File: proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md (hunk 1 above)
Comment (hwilli88) [reply proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md]: Done.

File: proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md (hunk 4 above)
Comment (hwilli88) [reply proposals/NNNN-Aligning-HMI-and-MOBILE-API-for-pcmStreamCapabilities.md]: Done

**Comment by hwilli88** [comment]
@theresalech Made changes per comments, ready for review.

**Pull request merged**